docs/Plex/what-does-my-media-player-support.md

```diff
@@ -0,0 +1,113 @@
+
+This allows for the best audio quality as the audio signal is not altered or compressed in any way before reaching the AV receiver or soundbar. This is especially important for high-quality audio formats such as Dolby TrueHD or DTS-HD Master Audio.
+
+By using audio passthrough, you can ensure that you are getting the best audio experience possible when watching movies or listening to music through your home theater system. It allows you to take full advantage of the capabilities of your AVR/Soundbar and ensures that you are hearing the audio as it was intended to be heard by the content creators.
+
+Overall, audio passthrough is the ideal way to send audio from your media server to your AV receiver or soundbar for the highest quality audio experience.
+
+- TrueHD Atmos
+- DTS X
+- DD+ Atmos
+- TrueHD
+- DTS-HD MA
+- FLAC/PCM
+- DD+ (E-AC3)
+- DTS
+- AAC
+- DD (AC3)
+
+## Link to tested Media Player Devices
+
+[Tested Media Players](https://docs.google.com/spreadsheets/d/15Wf_jy5WqOPShczFKQB28cCetBgAGcnA0mNOG-ePwDc/edit?usp=sharing){:target="_blank" rel="noopener noreferrer"}
+
+!!! info "A lot of results have a note added that provides extra information, such as fallback HDR or audio formats, as well as other information that we've gathered from several sources."
+
+---
+
+## What should I do next with this info
+
+The next thing to do is to setup a proper quality profile with the info you gathered from this sheet.
+To make this easier we made a How to setup a quality profile for Radarr/Sonarr that has a flowchart to help you even more to decide what to choose.
+
+### Radarr
+
+[How to set up Quality Profiles](/Radarr/radarr-setup-quality-profiles){:target="_blank" rel="noopener noreferrer"} - Explains how to make the most use of Custom Formats and shows some specifics of my configuration. You can use these examples to get an idea of how to set up your preferences.
+
+#### How to make use of HDR Formats for Radarr
+
+- [How to set up Quality Profiles](/Radarr/radarr-setup-quality-profiles){:target="_blank" rel="noopener noreferrer"} - Explains how to make the most use of Custom Formats and shows some specifics of my configuration. You can use
+
+- You have a 4K TV and a hardware media player device (such as Roku, AppleTV, Shield, SmartTV App, etc.) that supports several HDR formats (such as Dolby Vision, HDR10, HDR10+, etc.).
+
+{! include-markdown "../../includes/cf/radarr-all-hdr-formats.md" !}
+
+#### How to make use of Audio Formats for Radarr
+
+- [How to set up Quality Profiles](/Radarr/radarr-setup-quality-profiles){:target="_blank" rel="noopener noreferrer"} - Explains how to make the most use of Custom Formats and shows some specifics of my configuration. You can use
+
+- You have an audio setup that supports all HD Audio (TrueHD, DTS-X, DTS-HD, etc).
+- You have a media player (hardware devices) that can passthrough HD audio.
+- You don't have an AppleTV (Being it can't passthrough untouched HD audio).
```

Comment:
```suggestion
- You don't have an AppleTV (Being that it can't passthrough untouched HD audio).

Reply:
hmm, i think i will change this part, and you will need to re-review it